### DISCUSSION TITLE
Which branch of QEMU is used in the UTM (JIT) version 4.5.3 running on the iPhone(14 pro, iOS16.2)?

### DISCUSSION THREAD

**Discussion by huanggui** [utmapp/UTM#6627] [Development]
**Describe the issue**
I would like to know which branch of QEMU is used in the UTM (JIT) version 4.5.3 running on the iPhone. 

**Configuration (required)**
* UTM Version: UTM 4.5.3
* OS Version: iOS 16.2
* Device Model: iPhone 14 pro
* Is it jailbroken (name jailbreak used)? no
* How did you install UTM? Compiled and run through Xcode

**Comment by osy** [comment]
https://github.com/utmapp/qemu/tree/v7.2.0-utm

**Reply by huanggui** [reply]
Thank you for your answer. 
I have another question, why does enabling JIT make the system loading speed of UTM much faster compared to the SE version? What has UTM done with JIT, including any optimizations made by the dependent QEMU? Is it related to alloc_code_gen_buffer_splitwx_vmremap?
